szperacz/run.py

```diff
@@ -5,6 +5,9 @@
 from szperacz.files import FileHandler
 from tests import get_test_data_path
 
+import json
```

Comment:
Please optimize imports. In Pycharm: `Ctrl` + `Alt` + `O`

Further read: https://www.jetbrains.com/pycharm/guide/tips/optimize-imports/